scripts/commands/updateApiDocs.ts

```diff
@@ -278,9 +316,18 @@ async function convertHtmlToMarkdown(
       html,
       url: `${pkg.baseUrl}/${file}`,
       baseSourceUrl,
-      imageDestination: `/images/api/${pkg.name}`,
+      imageDestination: historical
+        ? `/images/api/${pkg.name}/${versionWithoutPatch}`
+        : `/images/api/${pkg.name}`,
       releaseNotesTitle: `${pkg.title} ${versionWithoutPatch} release notes`,
     });
+
+    result.markdown = transformLinks(result.markdown, (link, _) =>
+      !historical || /^\/images.*/.test(link) || !link.startsWith("http")
```

Comment:
What links are you trying to redirect and what are you not? For example, I think you might want `&& ~link.startswith("http")`, since those will never change; you're using `||`.

I recommend extracting this out to a `lib/` file and writing tests. It's important we get this right and don't break it.

Reply:
You are right, my intention was using the `||` but without the negation in front `link.startsWith("http")`.

Reply:
After our conversation, I removed this code because we don't have any absolute paths in the API docs, and therefore, this is not necessary anymore.